docs/reference/auditbeat/auditbeat-installation-configuration.md

```diff
@@ -253,6 +253,10 @@ Auditbeat comes with predefined assets for parsing, indexing, and visualizing yo
 
     `-e` is optional and sends output to standard error instead of the configured log output.
 
+
+By default, Windows log files are stored in `C:\Program Files\auditbeat-Data\logs`.
+For versions < 9.1.0, logs are stored by default in `C:\ProgramData\auditbeat\Logs`.
```

Comment:
This isn't right, the log path you use is `$LogsPath = "$HomePath\log"` which doesn't match these docs.

Reply:
That was a typo in the script, I've fixed it already. Thanks for catching it!